Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,30 +6,53 @@ class Program
     {
         static void Main(string[] args)
         {
-            Person();
-            Person();
+            PrintPersonInformation();
+            PrintPersonInformation();
         }
-
-        static void Person()
+        static string ReadName()
         {
             Console.WriteLine("Enter name");
             string name = Console.ReadLine();
+            return name;
+        }
+        static int ReadAge()
+        {
             Console.WriteLine("Enter age ");
             string age = Console.ReadLine();
-            int.Parse(age);
-            Console.WriteLine("Enter weight ");
+            int realAge = int.Parse(age);
+            return realAge;
+        }
+        public static double ReadWeight()
+        {
+            Console.WriteLine("Enter weight in kg ");
             string weight = Console.ReadLine();
-            double weightInDouble = double.Parse(weight);
+            double weightInKilograms = double.Parse(weight);
+            return weightInKilograms;
+        }
+        static double ReadHeight()
+        {
             Console.WriteLine("Enter height in cm ");
             string height = Console.ReadLine();
-            double heightInDouble = double.Parse(height);
-            double heightInMeters = heightInDouble / 100.0;
-
-            double BMI = weightInDouble / (heightInMeters * heightInMeters);
+            double heightInCentemeters = double.Parse(height);
+            return heightInCentemeters;
+        }
+        static double CalculateBMI(double weight, double height)
+        {
+            double heightInMeters = height / 100.0;
+            double BMI = weight / (heightInMeters * heightInMeters);
+            return BMI;
+        }
+        static void PrintPersonInformation()
```

Comment:
As mentioned before- higher level methods should be above lower level. The details (how it works), should be under the method.